File: exercicios/R/1564/1564.r
input = file('stdin','r')

while(TRUE){
    line = readLines(input,n=1)
    n = as.integer(line)

    if(length(line) == 0){
        break
    }else if(n == 0){
         write("vai ter copa!",'')
    }else if(n > 0){
        write("vai ter duas!",'')
    }
}

File: exercicios/JavaScript/1564/1564.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

while(lines.length > 1){
        var v = parseInt(lines.shift());
        
        if(v === null){
            break;
        }
        if(v === 0){
            console.log("vai ter copa!")
        }else{
            console.log("vai ter duas!")
        }
}